### ISSUE TITLE
libusb:x64-linux build failure

### ISSUE THREAD

**Issue by geoffreymbrown** [microsoft/vcpkg#19715]
**Host Environment**
- OS: linux
- Compiler: revision

Build failed because libudev headers are not installed.   I solved this by

sudo apt-get install libudev-dev

**Comment by autoantwort** [comment]
Yeah so there is no build failure. I think then you don't have create an issue for that :)

**Comment by geoffreymbrown** [comment]
You might say that, but I had to dig through the error messages to figure
out what went wrong.  So the error was indistinguishable from a build
failure.

**Comment by autoantwort** [comment]
It should be the first message printed by the port that you must install `libudev`

https://github.com/microsoft/vcpkg/blob/025e564979cc01d0fbc5c920aa8a36635efb01bb/ports/libusb/portfile.cmake#L4

**Comment by JonLiu1993** [comment]
@geoffreymbrown ,Does this problem still exist?
